### DISCUSSION TITLE
课程时间、大致安排、是否有录播

### DISCUSSION THREAD

**Discussion by guyiyifeurach** [mlc-ai/mlc-zh#6] [General]
如题，请主教公布下课程的具体时间？是在线播放，还是录播？

**Comment by jinhongyii** [comment]
美国时间17号开始，每周一次。录播

**Reply by zhizunbaowhy** [reply]
为啥中文主页写的不是北京时间，是不是中文主页的日程安排要更新成Beijing time

**Reply by Hzfengsy** [reply]
感谢建议，我们已经将中文主页的schedule改成了CST时区

**Reply by kun-teng** [reply]
美东时间17号吧？
录播挺好，不会错过: )
